support/ci/pipelines/docs.sh

```
@@ -3,6 +3,9 @@
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 source "$SCRIPT_DIR/_common.sh"
 
+DATE=$(git log -1 --format=%as)
+export COPYRIGHT_YEAR=$(date -d "$DATE" +%Y)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)